aspnet/mvc/models/validation/sample/Properties/launchSettings.json

```diff
@@ -0,0 +1,27 @@
+{
```

Comment:
Included this file for my own use: Otherwise IIS Express is the only option. Should I remove this boiler-plate?